lib/router/removeQueryParam.ts

```diff
@@ -0,0 +1,7 @@
+import type { NextRouter } from 'next/router';
+
+export default function removeQueryParam(router: NextRouter, param: string) {
+  const { pathname, query } = router;
+  delete router.query[param];
```

Comment:
I am not sure if it is a good idea to mutate the Nextjs router. Can we just call `router.replace()` with new query and let Nextjs do its job?

Reply:
the same comment to `lib/router/updateQueryParam.ts`